sunpy/map/tests/test_mapbase.py

```diff
@@ -82,7 +95,10 @@ def generic_map():
 
 def test_fits_data_comparison(aia171_test_map):
     """Make sure the data is the same in pyfits and SunPy"""
-    data = fits.open(os.path.join(testpath, 'aia_171_level1.fits'))[0].data
+    with warnings.catch_warnings():
```

Comment:
out of interest, why the `warnings.catch_warnings` here and not `pytest.warns`?

Reply:
We don't care about the warning?

Reply:
Um I'm not sure, let me get back to you on that one...

Reply:
Fixed